ci-lua

```diff
@@ -1,25 +1,43 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

Since you eliminated the Bash brackets in https://github.com/kaitai-io/kaitai_struct_tests/pull/89#discussion_r469587719, I think the shebang can be reverted back to POSIX shell for portability.